php/php.dbgp/nbproject/project.xml

```diff
@@ -180,7 +180,7 @@
                     <build-prerequisite/>
                     <compile-dependency/>
                     <run-dependency>
-                        <specification-version>2.87</specification-version>
+                        <specification-version>2.149</specification-version>
```

Comment:
Once this is done, typically also specification version of the module should be increased as well.

Why? If someone uses a daily build which is updated from DEV update center, he will get the new version of PHP project - that is fine. However, since PHP Debugger module has still the same specification version, no update is done. Therefore, functionality from this PR will not work. If the user downloads a fresh daily build, it will work. Therefore, the specification of the PHP Debugger module should be increased too so the user gets its updated version automatically too.

I am not sure where there is any DEV update center but in general, my comment applies.

Reply:
Thanks for the explanation.
I increased specification version of `php.dbgp` module.

Reply:
You are welcome.